src/views/pages/Identity/index.js

```diff
@@ -1,14 +1,21 @@
-import React, {useState, useContext} from 'react';
+import React, { useState, useContext } from 'react';
 import PageTitle from '../../components/PageTitle';
 import Text from '../../components/Text';
 import Button from '../../components/Button';
 import IdentitySubNav from '../../components/IdentitySubNav';
-import {useServiceContext} from '../../../contexts/ServiceContext';
+import { useServiceContext } from '../../../contexts/ServiceContext';
 
 function Identity() {
   const [createIdentity, setCreateIdentity] = useState(true);
   const [btnEnabled, setBtnEnabled] = useState(false);
-  const {keymanager, node} = useServiceContext();
+  const { keymanager, node } = useServiceContext();
+  const { fee, setFee } = useState(0);
+  const { balance, setBalance } = useState(0);
+
+  useEffect(() => {
+    setFee(node.getFeeForCreateIdentity(keymanager));
+    setBalance(node.getBalance(keymanager));
```

Comment:
both `getFeeForCreateIdentity` and `getBalance` return promises, not actual values. The set state methods require the actual value.